src/ts/AppBar/index.tsx

```diff
@@ -73,4 +79,6 @@ export default class AppBarOwn extends Component<any, any> {
   }
 }
 
-
+AppBarVG.childContextTypes = {
+  muiTheme: React.PropTypes.object.isRequired,
+};
```

Comment:
this is just for testing. The files are tested separately, so there is no wrapper around the tested components. We should most likely change that in the `test-setup` file, so that we don't have to do it everywhere. My intention was to get the tests up and running, and that goal was met.